src/main/java/seedu/dukeofbooks/data/book/Book.java

```diff
@@ -3,7 +3,7 @@
 import seedu.dukeofbooks.common.IVerifiable;
 import seedu.dukeofbooks.data.person.Person;
 
-public class Book implements IVerifiable {
+public class Book extends BorrowableItem implements IVerifiable {
```

Comment:
I think borrowableItem should be an interface and not a abstract class since it is like a property. Maybe can consider using it as a interface instead?